src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color, double side) {
```

Comment:
Consider creating separate variables for the 'color' and 'side' before passing them to the constructor for better readability, especially if the values are the result of expressions or method calls.